ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ImageMessage.qml

```diff
@@ -31,19 +31,24 @@ Rectangle {
         }
 
         Image {
+            // This minimum height is for the initial load of the chat to work correctly
+            // Without this, the chat doesn't understand that the image wil lhave a height and doesn't scroll to the bottom
+            property int minHeight: 400
             id: imageMessage
             anchors.horizontalCenter: parent.horizontalCenter
             anchors.top: (index == 0) ? parent.top: parent.children[index-1].bottom
             anchors.topMargin: imageChatBox.chatVerticalPadding
             sourceSize.width: imageChatBox.imageWidth
             source: modelData
             onStatusChanged: {
-                if (imageMessage.status == Image.Error) {
+                if (imageMessage.status === Image.Error) {
                     imageMessage.height = 0
+                    imageMessage.minHeight = 0
+                    imageMessage.source = ""
```

Comment:
On error, do you think it's a good idea if we replaced the image source with an error icon or something similar, to let the user know the image was not loaded?

Reply:
Good idea, I'll check if we have something like that in the design or icons